tests/lsan.supp

```diff
@@ -11,27 +11,59 @@ leak:qd_policy_open_fetch_settings
 leak:qdr_error_description
 
 # DISPATCH-1844 - shutdown leak
-leak:sys_mutex
+#leak:sys_mutex
 
 # expected, not a bug:
 #
-leak:qdr_core_subscribe
+#leak:qdr_core_subscribe
 
 # Ubuntu 16.04 (Xenial)
 #
 leak:_ctypes_alloc_format_string
 leak:__strdup
 
+# to be triaged; system_tests_http
+leak:^callback_healthz$
+leak:^callback_metrics$
```

Comment:
if I understand correctly, these leaks would be per-request, not single leak per run; but I did not investigate this yet

one of the leaks hidden by too-broad suppressions